src/classes/groundplaces.ts

```diff
@@ -377,27 +467,37 @@ export class GroundPlacesController {
     const cloneGroundPlaces: GroundPlace[] = cloneDeep(this.getGroundPlaces());
 
     try {
-      this.addStopGroupToStopCluster(stopGroupToMoveGpuid, intoStopClusterGpuid);
-      this.removeStopGroupFromStopCluster(stopGroupToMoveGpuid, fromStopClusterGpuid);
+      this.addStopGroupToStopCluster(stopGroupToMoveGpuid, intoStopClusterGpuid, CreateActionHistory.FALSE);
+      this.removeStopGroupFromStopCluster(stopGroupToMoveGpuid, fromStopClusterGpuid, CreateActionHistory.FALSE);
     } catch (error) {
       // If there is an error, previous update is reverted
       this.storageService.setGroundPlaces(cloneGroundPlaces);
 
       throw new Error(error.message);
     }
+
+    if (createActionHistory !== CreateActionHistory.FALSE) {
```

Comment:
weird ? Why not if (createActionHistory === CreateActionHistory.TRUE) ?

Reply:
Because in a precedent discussion, you and Theo discussed about don't use default value for the createActionHistory param. "This param should be only for purpose use" as you said, so in most times this is `undefined`.
If i do `(createActionHistory === CreateActionHistory.TRUE)`, I should use default value no?

Reply:
YES, we said NO DEFAULT value 
It means this should NEVER be undefined either  CreateActionHistory.TRUE OR CreateActionHistory.FALSE

Reply:
Ok done!